net-dns/knot-resolver/knot-resolver-5.7.4.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LUA_COMPAT=( luajit )
+
+inherit lua-single meson tmpfiles verify-sig
+
+DESCRIPTION="A scaleable caching DNS resolver"
+HOMEPAGE="https://www.knot-resolver.cz https://gitlab.nic.cz/knot/knot-resolver"
+SRC_URI="
+	https://knot-resolver.nic.cz/release/${P}.tar.xz
```

Comment:
```
>>> Downloading 'https://knot-resolver.nic.cz/release/knot-resolver-5.7.4.tar.xz'
--2024-09-23 12:30:56--  https://knot-resolver.nic.cz/release/knot-resolver-5.7.4.tar.xz
Resolving knot-resolver.nic.cz... 217.31.202.46, 2001:1488:ffff::46
Connecting to knot-resolver.nic.cz|217.31.202.46|:443... connected.
ERROR: no certificate subject alternative name matches
        requested host name 'knot-resolver.nic.cz'.
To connect to knot-resolver.nic.cz insecurely, use `--no-check-certificate'.
!!! Couldn't download 'knot-resolver-5.7.4.tar.xz'. Aborting.
```
Although I can't even download https://secure.nic.cz/files/knot-resolver/knot-resolver-5.7.4.tar.xz from their official web page, I need to try later.